app/models/user.rb

```diff
@@ -107,11 +107,22 @@ def gpo_verification_pending_profile?
   end
 
   def pending_profile
-    gpo_verification_pending_profile
+    profile = profiles.where(deactivation_reason: :in_person_verification_pending).or(
+      profiles.where.not(gpo_verification_pending_at: nil),
+    ).or(
+      profiles.where.not(fraud_review_pending_at: nil),
+    ).or(
+      profiles.where.not(fraud_rejection_at: nil),
```

Comment:
Someone with `fraud_rejection` is no longer "pending", although that might just be a terminology issue. If an account has been determined to be fraudulent by Fraud Ops, do we want to leave a pending_profile around?